### ISSUE TITLE
Tray icon has transparency issues

### ISSUE THREAD

**Issue by danielmeloalencar** [wnayes/bond-wm#7]
Hello! Firstly, congratulations on the incredible work!
I was investigating why the board is getting the primary color and I managed to come up with the following result, it's not a solution yet, but it could be a starting point for the solution:

![image](https://github.com/wnayes/bond-wm/assets/74947997/9c31efa8-15a2-4581-8415-68d1fbefbd18)

**Comment by danielmeloalencar** [comment]
nothing?

**Comment by wnayes** [comment]
I looked for quite a while, but I haven't been able to figure out how to get these tray icon windows to display with transparency. I can alter the background color to some other solid color though, so I can update it to match the taskbar color, which will look good for any solid colored taskbar. I will try to go with that approach for now since it's better than the current behavior.

**Comment by danielmeloalencar** [comment]
Yes, I believe it is the best solution for now.

**Comment by wnayes** [comment]
Sorry for the delay, I found some time to wrap up this change and publish a new minor version.

New packages with this change:

bond-wm@0.4.2
@bond-wm/shared@0.4.1
@bond-wm/react-taskbar@0.4.1

Closing this issue since the background color approach should be sufficient for any solid colored taskbar. If someone has a more complex styled taskbar with gradient/transparency this could be revisited in a new issue.
